cbindgen.toml

```diff
@@ -18,6 +18,9 @@ language = "C"
 
 [export]
 prefix = "WGPU"
+include = [
+    "AnisotropicSamplerDescriptorExt"
```

Comment:
why did you need to do this? Is the type skipped if not explicitly included?

Reply:
Yeah it wasn't explicitly included, let me peruse the cbindgen docs to see if there's some flag we need to throw.

Reply:
There appears to be no way to force include all applicable structs. It either needs to be in this list, or used by a function.

Reply:
Ah I see, that makes sense.